helpers/QueryHelper.php

```diff
@@ -214,6 +214,35 @@ public static function insert_multiple_rows( $table, $request ) {
 		return true;
 	}
 
+	/**
+	 * Make tge where clause base on its column operator and values.
+	 *
+	 * If the operator is IN then make the clause like `WHERE column_name IN (value1, value2, ...)`
+	 * Otherwise the clause would be `WHERE column_name = 'value'`
+	 *
+	 * @since 3.0.0
+	 *
+	 * @param array $where 	The where clause array. e.g. array( 'id', 'IN', array(1, 2, 3) ) or array( 'id', '=', 1 )
+	 *
+	 * @return string
+	 */
+	private static function make_clause( array $where ) {
```

Comment:
We already have `prepare_in_clause`

Reply:
Need to perform query like -
```
SELECT * FROM table_name WHERE order_id = 1 AND meta_key IN ('refunded', 'partially-refunded');
```

Is it possible by the current functionalities?

Reply:
No